app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -98,13 +99,13 @@ export default {
       const sortDatasets = this.items
       const allDatasets = sortDatasets.sort((a, b) => a.attributes.name.localeCompare(b.attributes.name));
       let slug = this.$route.params.id
-      const indexToggle = allDatasets.findIndex(dataset => dataset.attributes.slug == slug)
+      const indexToggle = allDatasets.findIndex(dataset => dataset.attributes.slug === slug)
       this.toggle = indexToggle
       if (this.toggle === -1) {
         this.toggle = 0
         slug = slug = allDatasets.length ? allDatasets[0].attributes.slug : ''
```

Comment:
what does `slug = slug = allDatasets`? I think one of the `slug =` is duplicated

Reply:
Fixed here: https://github.com/PopulateTools/gobierto/pull/2975/commits/d3bde171a5a52a31a5d0d6ad121161697abe0244

Reply:
Take into account the Github label `Outdated` ( ͡° ͜ʖ ͡°)